CHANGELOG.md

```diff
@@ -1,8 +1,34 @@
 ## Next release
 
+## 0.39.0 / 2020-05-06
+
+* [CHANGE] Introduce release schedule (#3135)
+* [CHANGE] Remove Custom Resource Definition management and move CRD definitions to bundle.yaml (#3155, #3171)
+* [CHANGE] Remove apiextensions.k8s.io/v1beta1 CRD and require kubernetes v1.16 or newer (#3175, #3187)
+* [FEATURE] Add support prometheus query log file (#3116)
+* [FEATURE] Add support for watching specified rules directory by config-relader (#3128)
+* [FEATURE] Add TLS support for operator web server (#3134, #3157)
+* [FEATURE] Allow to set address for operator http endpoint (#3098)
+* [FEATURE] Allow setting the alertmanagers cluster.advertiseAddress (#3160)
+* [FEATURE] Add support of matching only PodMonitors (#3173)
+* [FEATURE] Build operator images for ARM and ARM64 platforms (#3177)
+* [ENHANCEMENT] Allow setting log level and format for thanos sidecar (#3112)
+* [ENHANCEMENT] Support naming of remote write queues (#3144)
+* [ENHANCEMENT] Allow disabling mount subPath for volumes (#3134)
+* [ENHANCEMENT] Update k8s libraries to v1.18 (#3154)
```

Comment:
is it worth mentioning here?

Reply:
I was also thinking about it, but we did include such things in the past.

Reply:
it was breaking changes so maybe not enhancement, but those that use it as a library might need this

Reply:
In last releases we were mentioning such updates as `[ENHANCEMENT]`. We did so in 0.34.0 when we upgraded to 0.16.0, same went in for 0.37.0 when we updated to 0.17.3. I was merely following pattern and I am fine with removing this completely if you think it is not important.

Reply:
Works for me then.